apps/frontend/src/libs/modules/store/libs/middlewares/error-handling.middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { isRejectedWithValue, type Middleware } from "@reduxjs/toolkit";
+import { toast } from "react-toastify";
+
+import { type AsyncThunkRejectValue } from "~/libs/types/types.js";
+
+const middleware: Middleware = () => {
```

Comment:
let's call it handleError, and the file name should be `handle-error.middleware.ts`
```suggestion
const handleError: Middleware = () => {
```

Reply:
Sounds much better